app/data/models/completed-payment-request.js

```diff
@@ -24,7 +24,8 @@ module.exports = (sequelize, DataTypes) => {
     value: DataTypes.INTEGER,
     submitted: DataTypes.DATE,
     acknowledged: DataTypes.DATE,
-    settled: DataTypes.DATE,
+    lastSettlement: DataTypes.DATE,
+    settledValue: DataTypes.INTEGER,
```

Comment:
Is this guaranteed to be an int rather that float?

Reply:
Yes.  The `ffc-pay-responses` repo will take the decimal value from the actual return file and convert it to pence.